am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -12,39 +12,21 @@
 import java.util.Set;
 
 import static uk.gov.hmcts.reform.amlib.enums.SecurityClassification.PUBLIC;
-import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.ROLE_NAME;
 
-@SuppressWarnings("LineLength")
 public final class DefaultRoleSetupDataFactory {
 
     private DefaultRoleSetupDataFactory() {
         throw new UnsupportedOperationException("Constructing utility class is not supported");
     }
 
+    @SuppressWarnings("LineLength")
     public static Map<JsonPointer, Map.Entry<Set<Permission>, SecurityClassification>> createPermissionsForAttribute(JsonPointer attribute, Set<Permission> permissions, SecurityClassification securityClassification) {
         Map.Entry<Set<Permission>, SecurityClassification> pair = new Pair<>(permissions, securityClassification);
 
         return ImmutableMap.of(attribute, pair);
     }
 
-    public static DefaultPermissionGrant createDefaultPermissionGrant(ResourceDefinition resourceDefinition, Set<Permission> permissions) {
-        return createDefaultPermissionGrant(resourceDefinition, permissions, JsonPointer.valueOf(""), PUBLIC);
-    }
-
-    public static DefaultPermissionGrant createDefaultPermissionGrant(ResourceDefinition resourceDefinition, JsonPointer attribute, Set<Permission> permissions) {
-        return createDefaultPermissionGrant(resourceDefinition, permissions, attribute, PUBLIC);
-
-    }
-
-    public static DefaultPermissionGrant createDefaultPermissionGrant(ResourceDefinition resourceDefinition, Set<Permission> permissions, JsonPointer attribute, SecurityClassification securityClassification) {
-        return DefaultPermissionGrant.builder()
-            .roleName(ROLE_NAME)
-            .resourceDefinition(resourceDefinition)
-            .attributePermissions(createPermissionsForAttribute(attribute, permissions, securityClassification))
-            .build();
-    }
-
-    public static DefaultPermissionGrant createDefaultPermissionGrant(ResourceDefinition resource, Set<Permission> permissions, String attribute, String roleName) {
+    public static DefaultPermissionGrant createDefaultPermissionGrant(String roleName, ResourceDefinition resource, Set<Permission> permissions, String attribute) {
```

Comment:
Reorder args so that attribute comes before permissions.

Reply:
👍